app/models/concerns/conclusion_reviews/gal_pdf.rb

```diff
@@ -821,6 +821,7 @@ def show_scope_detail?
       !show_review_best_practice_comments?(organization) &&
         !collapse_control_objectives &&
         SCOPE_DETAIL_IN_CONCLUSION_REVIEW_START &&
-        created_at >= SCOPE_DETAIL_IN_CONCLUSION_REVIEW_START
+        created_at >= SCOPE_DETAIL_IN_CONCLUSION_REVIEW_START &&
+        review.plan_item.plan.period.start >= SCOPE_DETAIL_IN_CONCLUSION_REVIEW_START
```

Comment:
```suggestion
        review.period.start >= SCOPE_DETAIL_IN_CONCLUSION_REVIEW_START
```
Está relacionado directamente con el periodo (algo que se _ve_ en `app/models/review.rb`) y el otro ya no es necesario, porque de hecho pueden crear 2021 ahora y mejor que vayan todos con el mismo.